qiskit/__init__.py

```diff
@@ -112,6 +112,15 @@ def __bool__(self):
                 from qiskit.providers import ibmq
 
                 self.ibmq = ibmq.IBMQ
+                warnings.warn(
+                    "The qiskit.IBMQ entrypoint and the qiskit-ibmq-provider package ("
+                    "accessible from 'qiskit.providers.ibmq`) are deprecated and will be removed "
+                    "in a future release. Instead you should use the qiskit-ibm-provider package "
+                    "which is accesible from 'qiskit_ibm_provider'.",
```

Comment:
```suggestion
                warnings.warn(
                    "The qiskit.IBMQ entrypoint and the qiskit-ibmq-provider package ("
                    "accessible from 'qiskit.providers.ibmq`) are deprecated and will be removed "
                    "in a future release. Instead you should use the qiskit-ibm-provider package "
                    "which is accesible from 'qiskit_ibm_provider'. You can install it with 'pip install qiskit_ibm_provider'",
```

Reply:
Similarly, I suggest to add a sentence about pip install for all other deprecation warnings.

Reply:
Added in: https://github.com/Qiskit/qiskit-terra/pull/8080/commits/da1b5f0f9e410b620f16e52c04b4979d71861163